frontend/test/metabase/scenarios/embedding/embedding-linked-filters.cy.spec.js

```diff
@@ -237,6 +241,205 @@ describe("scenarios > embedding > dashboard > linked filters (metabase#13639, me
+        cy.findByText("Doohickey").should("not.exist");
+        cy.findByText("Gadget").should("not.exist");
+        cy.findByText("Widget").should("not.exist");
+
+        cy.button("Add filter").click();
+      });
+
+      cy.location("search").should("eq", "?id_filter=1&category=Gizmo");
+
+      cy.findByTestId("table-row")
+        .should("have.length", 1)
+        .and("contain", "Gizmo");
+    });
+
+    it("works when main filter's value is set through URL", () => {
+      cy.get("@guiDashboardId").then(dashboard_id => {
+        const payload = {
+          resource: { dashboard: dashboard_id },
+          params: {},
+        };
+
+        cy.log("Make sure we can override the default value");
+        visitEmbeddedPage(payload, { setFilters: "id_filter=4" });
+
+        cy.location("search").should("eq", "?id_filter=4");
+
+        filterWidget()
+          .should("have.length", 2)
+          .contains("Category")
+          .click();
+
+        popover().within(() => {
+          cy.findByText("Doohickey").click();
+          cy.findByText("Gizmo").should("not.exist");
+          cy.findByText("Gadget").should("not.exist");
+          cy.findByText("Widget").should("not.exist");
+
+          cy.button("Add filter").click();
+        });
+
+        cy.location("search").should("eq", "?id_filter=4&category=Doohickey");
+
+        cy.findByTestId("table-row")
+          .should("have.length", 1)
+          .and("contain", "Doohickey");
+
+        cy.log("Make sure we can set multiple values");
+        visitEmbeddedPage(payload, {
+          setFilters: "id_filter=4&id_filter=29&category=Widget",
+        });
+
+        filterWidget()
+          .should("have.length", 2)
+          .and("contain", "2 selections")
+          .and("contain", "Widget");
+
+        cy.findByTestId("table-row")
+          .should("have.length", 1)
+          .and("contain", "Widget")
+          .and("contain", "Durable Steel Toucan");
+
+        removeValueForFilter("Category");
+
+        cy.findAllByTestId("table-row")
+          .should("have.length", 2)
+          .and("contain", "Widget")
+          .and("contain", "Doohickey")
+          .and("contain", "Durable Steel Toucan");
+
+        cy.findByText("2 selections").click();
+
+        // Remove one of the previously set filter values
+        popover()
+          .findByText("29")
+          .closest("li")
+          .find(".Icon-close")
+          .click();
+        cy.button("Update filter").click();
+
+        cy.findByTestId("table-row")
+          .should("have.length", 1)
+          .and("contain", "Doohickey");
+
+        openFilterOptions("Category");
+
+        popover().within(() => {
+          cy.findByText("Doohickey");
+          cy.findByText("Gizmo").should("not.exist");
+          cy.findByText("Gadget").should("not.exist");
+          cy.findByText("Widget").should("not.exist");
+        });
+      });
+    });
+
+    it("works when the default filter is hidden", () => {
```

Comment:
Something's off in this test. Either I'm tripping on the test name, or the missing setting of the default.

Reply:
It's a bit tricky when test data is kept separately from the main test. I would have a lot of noise otherwise.
Here's the default value:
https://github.com/metabase/metabase/pull/21529/files/03944dd10d2dbcb212016289780e6d200b93f8ad#diff-2f97aa9c00438a0b62552ecc02787dba5bbe6e7277112253682915eaa23cec9eR93

And also a comment about it in this spec:
https://github.com/metabase/metabase/pull/21529/files/03944dd10d2dbcb212016289780e6d200b93f8ad#diff-7c90f1fbab9cc26397e5250d4bedd292d87051234dee89e665d4d859a472a42eR275

Reply:
@flamber would "works when the main filter with the default value is hidden" work better?

Reply:
Yeah, let's do that. I think both the title and the missing "enabled" in params (which I know is the default) tripped me.

Reply:
Both filters are enabled by default.
https://github.com/metabase/metabase/pull/21529/files/03944dd10d2dbcb212016289780e6d200b93f8ad#diff-7c90f1fbab9cc26397e5250d4bedd292d87051234dee89e665d4d859a472a42eR33-R40